doc/source/Python.rst

```diff
@@ -327,3 +380,10 @@ Through ``pygrackle``, the following ``yt`` fields are defined:
 These fields are created after calling the ``add_grackle_fields`` function.
 This function will initialize Grackle with settings from parameters in the
 loaded dataset. Optionally, parameters can be specified manually to override.
+
+
+.. rubric:: Footnotes
+
+.. [#f1] When you want to overwrite a flag in a CMake build, we generally encourage use of CMake-specific counterparts to environment variables (e.g. prefer passing ``-DCMAKE_C_COMPILER=<blah>`` on the command-line to exporting ``CC=<blah>``), since the former has precedence and is more case.
```

Comment:
"...and is more case." Not sure I understand what you mean here.

Reply:
Yeah, that made no sense. I reworded this.